packages/yew/src/format/nothing.rs

```diff
@@ -7,6 +7,7 @@ use anyhow::bail;
 #[derive(Debug)]
 pub struct Nothing;
 
+#[allow(clippy::from_over_into)]
```

Comment:
why didn't you change it here too?

Reply:
Because this error-ed out instead of creating an instance in `Into` whereas it actually created an instance in `From`. Let me know if this should be changed.

Reply:
ah well, let's just roll with it then.